### PR TITLE
fix: toggle time display on click from remaining / not remaining

### PR DESCRIPTION
**Pull request by dylanjha**
Discussion: https://github.com/muxinc/media-chrome/discussions/181

- [x] updates `<media-time-display>` behavior so that it toggles on click between current / remaining
- [x] handles keyboard interaction with Enter & Spacebar
- [x] Do we need `cursor: pointer:` here? What's the best place to put that style rule?
- [x] What's the right conventional commit here? `improv:` ? `fix:` ?

### CODE REVIEW

File: src/js/media-time-display.js [hunk 1]
```diff
@@ -4,6 +4,9 @@ import { formatAsTimePhrase, formatTime } from './utils/time.js';
 import { MediaUIAttributes } from './constants.js';
 import { nouns } from './labels/labels.js';
 // Todo: Use data locals: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleTimeString
+//
+
+const ButtonPressedKeys = ['Enter', ' '];
```
Comment (dylanjha) [inline src/js/media-time-display.js:9]: Should be somewhere else?

File: src/js/media-time-display.js [hunk 2]
```diff
@@ -71,9 +74,39 @@ class MediaTimeDisplay extends MediaTextDisplay {
 
     this.setAttribute('role', 'progressbar');
     this.setAttribute('aria-label', nouns.PLAYBACK_TIME());
+
+    const keyUpHandler = (evt) => {
```
Comment (dylanjha) [inline src/js/media-time-display.js:78]: seems like some boilerplate that I had to add here to handle keypresses, is there a more generic util for this?

File: src/js/media-time-display.js (hunk 1 above)
Comment (luwes) [reply src/js/media-time-display.js:9]: fine I think

File: src/js/media-time-display.js (hunk 2 above)
Comment (luwes) [reply src/js/media-time-display.js:78]: it's fine here I think, copy a little code

**Comment by luwes** [comment]
I'd go with `fix:` for this change, in the <v1.0.0 state. 
`cursor: pointer` could be an inline style on the span since it's in the shadow DOM

**Review by luwes — APPROVED** [review]
LGTM!

I'll follow up this with a few small changes around the `<slot>` logic of MediaTextDisplay,
it's weird that we remove the slot I think.

**Pull request merged**